aiida/engine/processes/calcjobs/tasks.py

```diff
@@ -179,11 +179,11 @@ async def do_update():
         if job_info is None:
             # If the job is computed or not found assume it's done
             node.set_scheduler_state(JobState.DONE)
-            job_done = True
         else:
             node.set_last_job_info(job_info)
             node.set_scheduler_state(job_info.job_state)
-            job_done = job_info.job_state == JobState.DONE
+
+        job_done = job_info.job_state == JobState.DONE
```

Comment:
This change will now raise if `job_info` is `None`, right? Don't see why this change is necessary

Reply:
Oops, thanks for spotting!

I guess this should have been
```
job_done = node.get_scheduler_state() == JonState.DONE
```
but then it's a database access, so I will revert this